docs/components/concepts/document-handling.md

```diff
@@ -95,23 +116,45 @@ To ensure seamless integration and functionality of document handling with GCP s
 
 <TabItem value='local'>
 
-| Store variable           | Description                                    |
-| ------------------------ | ---------------------------------------------- |
-| DOCUMENT_STORE_GCP_CLASS | The path where the documents should be stored. |
+:::note
+Local storage is not usable in production, and is not supported for Self-Managed installations via Helm charts.
+:::
+
+| Store variable             | Required | Description                                                                                                                                 |
+| -------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| DOCUMENT_STORE_LOCAL_PATH  | Yes      | The path to the directory which will host the uploaded files.                                                                               |
+| DOCUMENT_STORE_LOCAL_CLASS | Yes      | The class for instantiating the local store. This must always be `io.camunda.document.store.localstorage.LocalStorageDocumentStoreProvider` |
+
+**Example:**
+
+```
+DOCUMENT_STORE_LOCAL_CLASS=io.camunda.document.store.localstorage.LocalStorageDocumentStoreProvider
+DOCUMENT_STORE_LOCAL_PATH=/usr/local/camunda
+DOCUMENT_DEFAULT_STORE_ID=local
+```
 
 </TabItem>
 
 <TabItem value='in-memory'>
 
+:::note
+In-memory storage is not usable in production.
+:::
+
 If no configuration is provided and no `DOCUMENT_DEFAULT_STORE_ID` is set, in-memory is used as the default storage type.
 
 If configurations are provided for at least one storage type (DOCUMENT_STORE_AWS_BUCKET, DOCUMENT_STORE_AWS_BUCKET_PATH, etc.) ensure `DOCUMENT_DEFAULT_STORE_ID=in-memory` is set for in-memory storage.
```

Comment:
This reads a little ambiguously; I assume it means the following:

```suggestion
If configurations are provided for at least one storage type (DOCUMENT_STORE_AWS_BUCKET, DOCUMENT_STORE_AWS_BUCKET_PATH, etc.) `DOCUMENT_DEFAULT_STORE_ID=in-memory` must be set to enable in-memory storage.
```

I have the same question regarding `in-memory` here as below.